src/js/components/molecules/social-sharing/social-sharing.ts

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+import { Vue, Component, Prop } from 'vue-property-decorator';
+// import { SocialSharingProps } from './social-sharing.models';
```

Comment:
Why are you not using the props?

Reply:
There are no props… it's hardcoded… for now.

Reply:
Ok cool,l it not managed in the CMS, then please remove the line given it's not needed.